src/dgipy/network_graph.py

```diff
@@ -2,24 +2,38 @@
 
 import networkx as nx
 import pandas as pd
-import plotly.graph_objects as go
 
-PLOTLY_SEED = 7
+LAYOUT_SEED = 7
```

Comment:
this is probably a trivial detail, but is this a value that we would want to allow the user to set somehow (just in case)? Like the default can be this but they could potentially set an environmental variable or something if they really wanted to mess with it.

Reply:
I'm assuming this just helps determine the initial placement and orientation of the nodes

Reply:
Changing it doesn't really make any significant changes to the graph. The seed pretty much just randomizes the node positions and orientation of the graph. The only reason it has a specified value (in this case 7) is to ensure that querying the same drugs/genes twice will output identical graphs.

We certainly could allow users to change it, but it wouldn't really help them mess with the layout of the graph in any meaningful way.